src/app/modules/admin/components/new-department/new-department.component.html

```diff
@@ -1,8 +1,6 @@
 <div class="new-department">
   <a class="nav-link regular_text" routerLink="/admin/dashboard">
-    <svg xmlns="http://www.w3.org/2000/svg">
-      <path d="M4 1 L1 5 L4 9" />
-    </svg>
+    <img src="assets/svg/Vector-arrow-back.svg" alt="back" style='margin-right:15px'>
```

Comment:
С маленькой название файла
```suggestion
    <img src="assets/svg/vector-arrow-back.svg" alt="back" style='margin-right:15px'>
```